java/bin/start.sh

```diff
@@ -16,6 +16,28 @@ set -e
 # limitations under the License.
 
 #Initialize functions and Constants
+echo "Script Started Execution"
+java --version
+java_status=$?
+
+if test $java_status -eq 0
+then
+	printf "\n Java is installed, thus we are good to go \n"
+else
+	printf "\n Java is not installed on this machine, thus we need to install that first \n"
```

Comment:
If java is not installed you need to stop the execution on this script.
Otherwise there is no value in checking for it.

Reply:
Correspondingly made changes in the code.